python/README.rst

```diff
@@ -16,14 +16,31 @@ and please indicate which language SDK you're using in the report.
 Sample project setup
 --------------------
 
-In these instructions we'll use `pyenv <https://github.com/pyenv/pyenv#installation>`_
-to install python3.7 and
-`pipenv <https://docs.pipenv.org/en/latest/#install-pipenv-today>`_
-to manage project dependencies. Here is how to install them on a mac
+Install python 3.7. We highly recommend using
+`pyenv <https://github.com/pyenv/pyenv#installation>`_ to install
+different versions of python. Mac users should use
+`homebrew <https://brew.sh/>`_ to install pyenv:
 
 .. code-block:: bash
 
-    brew update && brew install pyenv && brew install pipenv
+    brew install pyenv
+
+Follow the **remaining steps 3 - 5** of
```

Comment:
Might be worth numbering the steps if we're referring to them like this.

Reply:
this is in reference to the link (https://github.com/pyenv/pyenv#basic-github-checkout) which does have the steps numbered :-)